Hazelcast.Examples/Transactions/TransactionBenchmark.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) 2008-2017, Hazelcast, Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using Hazelcast.Client;
+using Hazelcast.Config;
+using Hazelcast.Transaction;
+
+namespace Hazelcast.Examples.Transactions
+{
+    internal class TransactionBenchmark
+    {
+        public static void Run(string[] args)
+        {
+            Environment.SetEnvironmentVariable("hazelcast.logging.level", "info");
+            Environment.SetEnvironmentVariable("hazelcast.logging.type", "console");
+
+            var config = new ClientConfig();
+            config.GetGroupConfig().SetName("dev").SetPassword("dev-pass");
+            config.GetNetworkConfig().AddAddress("127.0.0.1:5701")
+                .SetSmartRouting(false);
```

Comment:
Why do we set the benchmark to use unisocket client? Somebody may try this sample and assume that this is as fast as it gets.

Reply:
Transactions use single connection so it does not scale I guess but I can update the name or add a comment